roles/check_config_static/tasks/main.yml

```diff
@@ -27,6 +27,13 @@
   fail: msg="One, or more tikv hosts may be specified."
   when: "groups.get('tikv_servers', [])|length < 1"
 
+- name: Ensure enable_tls is disabled when deployment_method is docker
+  fail: msg="TLS is not supported yet when deployment_method is docker in tidb-ansible."
+  when:
+    - deployment_method == 'docker'
+    - enable_tls is defined
+    - enable_tls
```

Comment:
Maybe can use `enable_tls|default(false)` instead of `enable_tls is defined && enable_tls`